### PR TITLE
Update service, services controllers to use getPreferredVersion

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Open TODO related to https://github.com/openshift/origin-web-common/pull/250.

### CODE REVIEW

**Comment by jwforres** [comment]
/hold

Would rather wait to merge till the TODO is resolved

**Comment by benjaminapetersen** [comment]
Sounds good.  Soon as we merge that PR in common I can resolve it.

**Comment by benjaminapetersen** [comment]
TODO resolved, rebased

**Comment by spadgett** [comment]
@benjaminapetersen dist mismatch

**Comment by benjaminapetersen** [comment]
rebased/rebuilt

File: app/views/browse/service.html [hunk 1]
```diff
@@ -100,7 +100,7 @@ <h3>Pods</h3>
                 <annotations annotations="service.metadata.annotations"></annotations>
               </div>
             </uib-tab>
-            <uib-tab active="selectedTab.events" ng-if="'events' | canI : 'watch'">
+            <uib-tab active="selectedTab.events" ng-if="eventsVersion | canI : 'watch'">
```
Comment (spadgett) [inline app/views/browse/service.html]: Maybe I missed it, but I don't see where `$scope.eventsVersion` is set?

Comment (benjaminapetersen) [reply app/views/browse/service.html]: bah, my bad.  thx, will update & retest.

**Comment by benjaminapetersen** [comment]
rebased & updated, thx @spadgett

**Comment by benjaminapetersen** [comment]
rebased

File: app/views/browse/service.html (hunk 1 above)
Comment (spadgett) [reply app/views/browse/service.html]: I still don't see `eventsVersion` set?

Comment (benjaminapetersen) [reply app/views/browse/service.html]: push your changes Ben... sorry, didn't push that last update.  This looks right now.

**Pull request merged**